tests/e2e/cucumber/steps/ui/resources.ts

```diff
@@ -562,17 +562,18 @@ Then(
 )
 
 Then(
-  '{string} should see {int} resources in the personal space files view',
+  '{string} should see {int} resources in the personal/project space files view',
   async function (this: World, stepUser: string, expectedNumberOfResources: number) {
     const { page } = this.actorsEnvironment.getActor({ key: stepUser })
     const resourceObject = new objects.applicationFiles.Resource({ page })
-    const actualNumberOfResources = await resourceObject.getNumberOfResourcesInThePage()
-    await expect(actualNumberOfResources).toBe(expectedNumberOfResources)
+    await resourceObject.expectNumberOfResourcesInThePageToBe({
+      numberOfResources: expectedNumberOfResources
+    })
```

Comment:
we should only get the required information from objects and assert here in step def
```suggestion
    const actualNumberOfResources = await resourceObject.getNumberOfResourcesInThePage()
    await expect(actualNumberOfResources).toBe(expectedNumberOfResources)
```